2 - RTB/Documentazione interna/Verbali/Interni/23-11-10/23-11-10.typ

```diff
@@ -0,0 +1,52 @@
+#import "/template.typ": *
+
+// Take a look at the file `template.typ` in the file panel
+// to customize this template and discover how it works.
+#show: project.with(
+  title: "Verbale 10/11/23",
+  subTitle: "Meeting di retrospettiva, indicazioni per la settimana a venire",
+  docType: "verbale",
+  //  Nome        //Ruolo           Durata Presenza
+  partecipants: (
+    (name: "Antonio", role: "Responsabile"),
+    (name: "Alessio", role: "Verificatore"),
+    (name: "Giovanni",  role: "Amministratore"),
+    (name: "Silvio",   role: "Redattore"),
+    (name: "Mattia",   role: "Amministratore"),
+    (name: "Rosario",  role: "Amministratore"),
+    (name: "Riccardo",  role: "Amministratore"),
+  ),
+  oraInizio: "14:00",
+  oraFine: "15:00"
+)
+
+#set text(font:"New Computer Modern", size: 1.3em)
+#set align(left)
+
+= Retrospettiva
+
+  In questo primo meeting di retrospettiva sono state analizzate le attività svolte dal gruppo durante la settimana e, in modo particolare, sono state discusse le nuove automazioni di github action che sono state scritte (per la compilazione ed il caricamento automatico in repository e nel sito web dei file legati alla documentazione con i relativi changelog).
+  \ Si è parlato del cambiamento che abbiamo attutato sulle tecnologie di markup per la stesura dei documenti passando da Latex a Typst dato che quest'ultimo ci è risultato più comodo da utilizzare.\
+  È stato analizzato il diario di bordo che verrà esposto lunedì 13 novembre, soffermandoci sulle difficoltà riscontrate.\ Nel fine settimana è importante che ciascuno pensi ad eventuali domande da aggiungere.
```

Comment:
~soffermandoci~ -> in particolar modo riguardo le difficoltà riscontrate (utilizzare forme impersonali). Ciascuno -> ciascun membro del gruppo